### ISSUE TITLE
Targeting planet only links to ships in the planet's SoI

### ISSUE THREAD

**Issue by Starstrider42** [RemoteTechnologiesGroup/RemoteTech#14]
Original issues at Cilph/RemoteTech2#173, Cilph/RemoteTech2#200.

krenshala wrote: 
> Also of note, targetting a specific bodies does not establish communication with the active vessel when outside that bodies SoI, even if the active vessel is inside the LoS angle. As an example, I set up a comm array with three dishes targetting the Mun, then sent a manned orbiter to the Mun on a free return trajectory. While on the line between the Mun and Kerbin, but outside the Mun's SoI, there was no communication despite being at the focus of three different satellites targetting the Mun. Comms immediately picked up as soon as the craft reached the Mun's SoI.

maxdreamland wrote:
> I stumbled upon a scenario where the function "isTargetingPlanet()" in RangeModelExtensions does not provide the correct return value.
> 
> the function asumes that a potentially targeted satelite has to orbit the targeted planet. only after this is checked it is tested if the satelite is inside the DishAngle ("vision cone"). This is not always sufficient.

**Comment by Peppie23** [comment]
Merged into v1.6
